IdentityCore/src/MSIDOAuth2Constants.h

```diff
@@ -120,7 +120,10 @@ extern NSString *const MSID_LAST_MOD_TIME_CACHE_KEY;
 extern NSString *const MSID_LAST_MOD_APP_CACHE_KEY;
 
 extern NSString *const MSID_ACCESS_TOKEN_CACHE_TYPE;
+extern NSString *const MSID_ACCOUNT_CACHE_TYPE;
 extern NSString *const MSID_REFRESH_TOKEN_CACHE_TYPE;
+extern NSString *const MSID_APPLICATION_METADATA_CACHE_TYPE;
+extern NSString *const MSID_ACCOUNT_METADATA_CACHE_TYPE;
```

Comment:
If some of the keys are Mac specific, will it be better to add a little bit of comments to it :)